cmd/server.go

```diff
@@ -242,6 +243,9 @@ var stringFlags = map[string]stringFlag{
 		description:  "Log level. Either debug, info, warn, or error.",
 		defaultValue: DefaultLogLevel,
 	},
+	LogPrefixToStrip: {
+		description: "Prefix used by Terraform client when determining if a log line should be stripped from the output.",
```

Comment:
Based on the code elsewhere looks like this is a comma delimited string of multiple prefixes but the name and description doesn't reflect that.

I also want us to use the global cfg where we can, cli flag for this seems like a pain if there was like 10 of these in a list.  In yaml, specifying a list is a piece of cake.